composer/trainer/ddp.py

```diff
@@ -79,6 +93,16 @@ def ddp_sync_context(state: State, is_final_microbatch: bool, sync_strategy: Uni
 
 
 def prepare_ddp_module(module: Model, find_unused_parameters: bool) -> Model:
+    """Wraps the module in a :class:`~torch.nn.parallel.DistributedDataParallel` object if
```

Comment:
```suggestion
    """Wraps the module in a :class:`torch.nn.parallel.DistributedDataParallel` object if
```

long paths preferred for torch-based ones for clarity